src/main/java/com/example/gruppajava/Service/ParkingEventServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.example.gruppajava.Service;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import org.springframework.stereotype.Service;
+
+import com.example.gruppajava.entity.ParkingEvent;
+import com.example.gruppajava.repository.CarRepository;
+import com.example.gruppajava.repository.ParkingEventRepository;
+import com.example.gruppajava.repository.UserRepository;
+
+@Service
+public class ParkingEventServiceImpl implements ParkingEventService {
+    private final UserRepository userRepository;
+    private final CarRepository carRepository;
+    private final ParkingEventRepository parkingEventRepository;
+
+    public ParkingEventServiceImpl(
+        UserRepository userRepository,
+        CarRepository carRepository,
+        ParkingEventRepository parkingEventRepository
+    ) {
+        this.userRepository = userRepository;
+        this.carRepository = carRepository;
+        this.parkingEventRepository = parkingEventRepository;
+    }
+
+    @Override
+    public ParkingEvent addParkingEvent(Map<String, String> body) {
+        Long userId = Long.parseLong(body.get("userId"));
+        Long carId = Long.parseLong(body.get("carId"));
+        int parkingslotId = Integer.parseInt(body.get("parkingslotId"));
```

Comment:
parkingslotId is of Long type not Integer

Reply:
You are looking at the wrong commmit,
check https://github.com/CosmicJay2k/grupp-a-java/pull/32/commits/1d4f1788d4441ff07d051ade323e038d529c3847

Reply:
OK, I found it. 
Sorry and thanks